language/vm/vm_runtime/src/txn_executor.rs

```diff
@@ -129,7 +129,7 @@ where
         let lhs = try_runtime!(self.execution_stack.pop_as::<T>());
         let result = f(lhs, rhs);
         if let Some(v) = result {
-            self.execution_stack.push(v);
+            self.execution_stack.push(v)?;
```

Comment:
This adds an unreachable code path unless you dramatically lower the allowed stack size.